finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
+
+1. Send an email to astropy-finance-committee@googlegroups.com with the following information:
```

Comment:
@kelle 's comment below alludes to this, but... how far in advance? Does it matter? Presumably we want to encourage submission in good time, both so we have plenty of time to discuss it and so that airfares etc don't climb too high.